api_volontaria/apps/position/test_view_positions.py

```diff
@@ -0,0 +1,242 @@
+    UserFactory,
+    AdminFactory,
+)
+from api_volontaria.testClasses import CustomAPITestCase
+
+import pytz
+from django.conf import settings
+LOCAL_TIMEZONE = pytz.timezone(settings.TIME_ZONE)
+
+
+class PositionsTests(CustomAPITestCase):
+
+    ATTRIBUTES = [
+        'id',
+        'url',
+        'name',
+        'description',
+        'hourly_wage',
+        'weekly_hours',
+        'minimum_duration_commitment',
+        'is_remote_job',
+        'is_posted',
+    ]
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.user = UserFactory()
+        self.user.set_password('Test123!')
+        self.user.save()
+
+        self.user2 = UserFactory()
+        self.user2.set_password('Test123!')
+        self.user2.save()
+
+        self.admin = AdminFactory()
+        self.admin.set_password('Test123!')
+        self.admin.save()
+
+        self.position = Position.objects.create(
+            hourly_wage=14,
+            weekly_hours=15,
+            minimum_duration_commitment=3,
+            is_remote_job=True,
+            is_posted=True,
+        )
+
+        self.position2 = Position.objects.create(
+            hourly_wage=14.5,
+            weekly_hours=30.5,
+            minimum_duration_commitment=6.5,
+            is_remote_job=False,
+            is_posted=False,
+        )
+
+    def test_create_new_position_as_admin(self):
+        """
+        Ensure we can create a new position if we are an admin.
+        """
+        data_post = {
+            "name": "My new position name",
+            "description": "My new position description",
+            "hourly_wage": 15,
+            "weekly_hours": 40,
+            "minimum_duration_commitment": 6,
+            "is_remote_job": True,
+            "is_posted": True,
+        }
+
+        self.client.force_authenticate(user=self.admin)
+
+        response = self.client.post(
+            reverse('position-list'),
+            data_post,
+            format='json',
+        )
+
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.check_attributes(content)
+
+    def test_create_new_position(self):
+        """
+        Ensure we can't create a new position if we are a simple user.
+        """
+        data_post = {
+            "name": "My new position name",
+            "description": "My new position description",
+            "hourly_wage": 15,
+            "weekly_hours": 40,
+            "minimum_duration_commitment": 6,
+            "is_remote_job": True,
+            "is_posted": True,
+        }
+
+        self.client.force_authenticate(user=self.admin)
+
+        response = self.client.post(
+            reverse('position-list'),
+            data_post,
+            format='json',
+        )
+
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.check_attributes(content)
+
```

Comment:
The description of the test seems correct, but the test by itself seems not accurate.

 - It should be a user
 - You should have an error and not a `HTTP_201_CREATED`